cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -20,4 +20,19 @@ Existing tools such as `oc` are more operations-focused and require a deep under
 * Official full support of Node.js and Java components.
 * Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the {product-title} cluster.
\ No newline at end of file
+* Lists all the available components and services from the {product-title} cluster.
+
+== Core concepts
+
+Project::
```

Comment:
@boczkowska please add articles to the beginning of all these definitions: A project comprises your...; An application...; A component...; A service

Reply:
Changed in https://github.com/openshift/openshift-docs/pull/18477/commits/5d462b2f0bd071a4900134053b985c98836d119d